### ISSUE TITLE
Franz randomly changes form any service to mircorsoft teams

### ISSUE THREAD

**Issue by Artofaj** [meetfranz/franz#1832]
First of all I want to thank you guys for this really nice piece of software. I have seen a lot of hate here you sure don't deserve.  Now, to the bug:

I am running Franz v5.4.1 on Manjaro. Every now and then, while I am typing a message in any service (FB messenger, whatsapp) the entry randomly loses focus, meaning that I the cursor disappears from the input field. I am able to type and the text doesn't appear anywhere. At first I thought this was a problem with my window manager, but then I realized these messages have been input every time into my active chat in microsoft teams. This means, that in more than one occasion I have spammed my working colleagues with my personal messages.

Since this happens under different conditions, I can't really provide a lot of information as to how to reproduce this error.

The bug has been around since the last update. I can't confirm that this bug wasn't present before, since I just started using microsoft teams around the time the last version of Franz was released.

I hope you find a way and time to look into this.

**Comment by pdsrebelo** [comment]
+1 on this issue, sometimes it's really annoying and end up writing stuff in the wrong channel :(

**Comment by KarlErfle** [comment]
I'm having this same issue. I will lose focus from any other service and it will end up in Teams.

**Comment by promarcel** [comment]
I'm expecting this issue also but in a different way. I am using Franz 5.5.0-beta.4.

When typing inside Slack or WhatsApp I lost just the focus of the input, no command keys pressed. But it switch not to another service. I expect because I didn't have Microsoft Teams there.

**Comment by BlietZ** [comment]
Also experiencing this issue. A fix would be very welcome :)

**Comment by ovelar** [comment]
I'm experiencing the same issue as @Artofaj, also on Manjaro using i3wm - I have not experienced the same on MacOS. This happens mostly when typing in Slack, and always ends up in MS Teams.

**Comment by johanrfbjorklund** [comment]
+1 

I experience the same as other reported here. I'm sitting on Windows 10 and using Franz 5.5.0-beta.4.

Though i cannot trace where my input goes after it looses focus, I do have MS Teams installed among other messaging apps like Messenger, Slack, LinkedIn.

**Comment by warrendt** [comment]
+1

**Comment by keckstrm** [comment]
I am experiencing the same problem, I always end up in teams. Quite annoying to be frank

**Comment by yannik995** [comment]
I also have the problem and always end up typing in teams but still see the actual service.

**Comment by Serjster** [comment]
Yes, this is a super annoying problem! Sometimes I've almost posted some content that could have serious consequences since my teams is my main business messaging app! I have almost stopped using Franz because of this! Please! This is an urgent bug!!!

**Comment by borospeti** [comment]
+1
I'm also experiencing the issue (Teams silently stealing focus from another service) with version 5.6.1 of Franz on Linux. Very annoying indeed.

**Comment by gitloc42** [comment]
+1
this is really annoying. sometimes you get a dataprotection problem if you send personal data to the wrong people :(
client: franz 5.6.1 under windows 10 64bit

**Comment by johnheadlee** [comment]
bump. Anyone looking at this??

**Comment by gitloc42** [comment]
i dont think so. maybe franz is scanning our inputs and this causes this "bug". maybe more a feature for somebody...
i will switch to rambox as good franz alternative.

**Comment by warrendt** [comment]
> i dont think so. maybe franz is scanning our inputs and this causes this "bug". maybe more a feature for somebody...
> i will switch to rambox as good franz alternative.

I have also switched to Rambox

**Comment by Artofaj** [comment]
I switched to Ferdi. That's basically Franz but in purple.

**Comment by maxjosino** [comment]
It's sad to know the bug has been around for a while without any fix, as I'm experiencing the same almost two years later the OP.

Client: Franz 5.7.0 (5.7.0.1852) under macOS Monterey 12.2 64

**Comment by LuggLD** [comment]
I've been experiencing the same issue for a few days or weeks now, but the focus always switches to one of the Slack services I have added.

Franz 5.8.0-beta1 on Windows 10.

**Comment by keckstrm** [comment]
Yeah I switched to fredi instead. I couldn't deal with Franz anymore, it was pissing me off.

**Comment by Wegerich** [comment]
I am having the same problem. I don't have MS Teams. The text mostly ends up in a slack window (my second workspace) but occasionally others.

**Comment by lucananni93** [comment]
Happens the same to me!
